src/components/Header.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ReactComponent as RequestIcon } from '../icons/request.svg';
+import styles from './Header.module.css';
+
+function Header() {
+  return (
+    <header className={styles.header}>
+      <div className={styles.iconBox}>
+        <RequestIcon />
+      </div>
+      <h2>Requests</h2>
```

Comment:
```suggestion
      <h1>Requests</h1>
```

let's change to h1 just due to semantic reasons, this is kinda the principal title of the page and will change as you can see here: https://github.com/dielduarte/graphql-devtools/issues/6#issuecomment-607613212. I think you can just accept the suggestion, thanks!

Reply:
gonna merge it by myself, cause I will need Header to work on settings module @StanleySathler !

Reply:
No worries, buddy :rocket: Well noticed!